www/configurator/src/index.js

```diff
@@ -77,18 +77,24 @@ Promise.all(configuratorInit).then(responses => {
   cssTranspiler = new CssTranspiler(responses[1], responses[0]);
 
   // Apply initial styles
-  const initialStyles = cssTranspiler.getCssWithVars(getUrlCssVars());
-  iframeManager.setStyle(initialStyles);
+  cssTranspiler.getCssWithVars(getUrlCssVars()).then(initialStyles => {
+    iframeManager.setStyle(initialStyles);
 
-  // Listen to has change events on the page
-  window.addEventListener('hashchange', handleHashChange_);
+    // Listen to has change events on the page
+    window.addEventListener('hashchange', handleHashChange_);
+  });
 });
 
 /**
  * Function to handle URL hash changes. This is not used until the initialization of the configurator is completed.
  *    Also, this will simply get the latest URL params, pass them to the transpiler, and set the styles in the iframe manager.
  */
 function handleHashChange_() {
-  const updatedStyles = cssTranspiler.getCssWithVars(getUrlCssVars());
-  iframeManager.setStyle(updatedStyles);
+  cssTranspiler.getCssWithVars(getUrlCssVars()).then(updatedStyles => {
+    iframeManager.setStyle(updatedStyles);
+  }).catch(() => {
+    // Set styles back to the initial styles
+    // TODO (torch2424 && camelburrito) - Talk offline to decide how to display transpilation error to user
+    iframeManager.setStyle(cssTranspiler.templateCss);
```

Comment:
So would you like me to just do nothing in the catch block? Since the styles aren't applied unless the worker resolves successfully